elastic/kibana
# [Fleet] Hide from UI some shipper options

Closes https://github.com/elastic/kibana/issues/147613 
Closes https://github.com/elastic/kibana/issues/147613

## Summary
Follow up of https://github.com/elastic/kibana/pull/145755

Introduce a feature flag (`showExperimentalShipperOptions`)  to hide from the UI the shipper options that are still not ready to be shown to the user. The only available parameters for now should be the following, as per [comment](https://github.com/elastic/ingest-dev/issues/1512#issuecomment-1409233389):

- Mem queue
- Max batch size
- Queue flush timeout

### Testing steps
-  Add a new output and in the  yaml editor add one of the following and save it;

```
shipper: {}
```
or

```
shipper:
  enabled: true
```

- Edit the output, the shipper section should appear under "advanced options"

<img width="742" alt="Screenshot 2023-02-15 at 16 30 01" src="https://user-images.githubusercontent.com/16084106/219096849-cf6041c5-654c-404d-b8de-7fdc13cec2f4.png">

- Generated full agent policy:
<img width="762" alt="Screenshot 2023-02-16 at 15 05 20" src="https://user-images.githubusercontent.com/16084106/219386500-6b791c33-afee-4458-af5e-3ffb381baf8c.png">


There are more parameters visible, but they're set to `null`

### Checklist
